client/src/lib/error-telemetry.js

```diff
@@ -15,27 +15,42 @@ class ErrorTelemetry {
     this.options = options;
   }
 
+  authenticate(uuid, token) {
+    logger.info(`Error telemetry: uuid=${uuid}, token=${token}`);
+    this.uuid = uuid;
+    this.token = token;
+    return this;
+  }
+
   /**
-   * (Private) default behaviour for the output where to send data to when the watched logging file
-   * has a modification detected.
-   */
-  callErrorTelemetryService(app, logDataObject) {
+  * (Private) default behaviour for the output where to send data to when the watched logging file
+  * has a modification detected.
+  */
+  callErrorTelemetryService(app, logDataObject, uuid, token) {
```

Comment:
I'm not sure I understand why this is being called with attributes that are already members of the objects? Why isn't the function referring to `this.uuid` and `this.token`?